packages/arb-token-bridge-ui/src/util/withdrawals/fetchWithdrawalsFromSubgraph.ts

```diff
@@ -15,7 +15,7 @@ export type FetchWithdrawalsFromSubgraphResult = {
   l2TxHash: string
   l2BlockNum: string
   direction: 'deposit' | 'withdrawal'
-  source: 'subgraph' | 'event_logs'
+  source: 'subgraph' | 'event_logs' | 'local_storage_cache'
```

Comment:
NIT: Hmm not sure why we have the 3 different sources when the type is `FetchWithdrawalsFromSubgraphResult` - cleary a result from subgraph.

Reply:
you're right, updated to 'subgraph' only